android/src/app/organicmaps/maps/MwmActivity.java

```diff
@@ -1,4 +1,4 @@
-package com.mapswithme.maps;
+package app.organicmaps.maps;
```

Comment:
Files like this one can be in app/organicmaps dir. What are the benefits of keeping them in additional maps subdir?